app/supabase/functions/planqtn_job/index.ts

```diff
@@ -197,17 +193,6 @@ Deno.serve(async (req) => {
           "run", // action
           "--job-type",
           jobRequest.job_type, // job type
-          "--task-uuid",
```

Comment:
I'm puzzled by this removal, it completely breaks the flow, the job requires the task store information to report back the result of the calculation and it does not have the ability to take env vars instead.